www/js/search.js

```diff
@@ -651,7 +651,19 @@ module.exports = {
     const $shabadList = this.$shabad || document.getElementById('shabad');
     $shabadList.innerHTML = '';
     banidb.loadCeremony(ceremonyID).then(rowsDb => {
-      const rows = rowsDb[0].Verse ? rowsDb.map(row => row.Verse) : rowsDb;
+      const rows = rowsDb.map(rowDb => {
```

Comment:
probably need to do something about overlay as well
![image](https://user-images.githubusercontent.com/2982784/57714149-bccf9180-7628-11e9-881c-b1753532aaae.png)